pkg/awsManager/vpcVpnManager.go

```diff
@@ -35,111 +37,124 @@ func ListVPCforDeletion(client clientpkg.Client) ([]*string, error) {
 			errFlag = true
 		}
 		// clear out all ELB
-		err = DeleteELB(client, vpcId)
+		err = deleteELB(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete ELBs")
 			errFlag = true
 		}
 		// clear out all network load balancer
-		err = DeleteNetworkLoadBalancer(client, vpcId)
+		err = deleteNetworkLoadBalancer(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete Network Load Balancers")
 			errFlag = true
 		}
 		// delete NAT gateway
-		err = DeleteNatgateway(client, vpcId)
+		err = deleteNatgateway(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete Nat Gateway")
 			errFlag = true
 		}
 		// detach and delete network interface
-		err = DetachAndDeleteNetworkInterface(client, vpcId)
+		err = detachAndDeleteNetworkInterface(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to detach and delete Network Interface")
 			errFlag = true
 		}
 		//detach any internet gateway , and delete it
-		err = DeleteGateway(client, vpcId)
+		err = deleteGateway(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete Gateway")
 			errFlag = true
 		}
 		//detach VPN gateway
-		err = DetachVpnGateway(client, vpcId)
+		err = detachVpnGateway(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to detach VPN Gateway")
 			errFlag = true
 		}
 		//cleaning network ACL
-		err = DeleteNetworkAcl(client, vpcId)
+		err = deleteNetworkAcl(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete Network ACL")
 			errFlag = true
 		}
 		//now cleaning route tables
-		err = DeleteRouteTables(client, vpcId)
+		err = deleteRouteTables(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete Route Tables")
 			errFlag = true
 		}
 		// now cleaning subnets associated with that vpc id
-		err = DeleteSubnetsForVPC(client, vpcId)
+		err = deleteSubnetsForVPC(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete Subnet for VPCs")
 			errFlag = true
 		}
 		//now cleaning security groups
-		err = DeleteSecurityGroups(client, vpcId)
+		err = deleteSecurityGroups(client, vpcID, logger)
 		if err != nil {
+			logger.Error(err, "Failed to delete security groups")
 			errFlag = true
 		}
 
-		_, err = client.DeleteVpc(&ec2.DeleteVpcInput{VpcId: vpcId})
+		output, err := client.DeleteVpc(&ec2.DeleteVpcInput{VpcId: vpcID})
 		if err != nil {
+			logger.Error(err, "Failed to delete VPC", output)
 			errFlag = true
-			fmt.Println("ERROR deleting VPC :", *vpcId)
-			fmt.Println(err)
 		}
 	}
 
 	if errFlag == true {
 		return ErrVpcNotDelete
 	}
-	fmt.Println("VPC's deleted successfully for this region  ")
+	logger.Info("VPC's deleted successfully")
 	return nil
-
 }
 
-func DeleteELB(client clientpkg.Client, vpcID *string) error {
-
+func deleteELB(client clientpkg.Client, vpcID *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file